code/datums/status_effects/neutral.dm

```diff
@@ -143,3 +143,81 @@
 /datum/status_effect/charging
 	id = "charging"
 	alert_type = null
+
+#define LWAP_LOCK_CAP 10
+
+/datum/status_effect/lwap_scope
+	id = "lwap_scope"
+	alert_type = null
+	duration = -1
+	tick_interval = 4
+	var/locks = 0
+	var/owner_dir = 0
+
+/datum/status_effect/lwap_scope/on_creation(mob/living/new_owner, stored_dir = 0)
+	owner_dir = stored_dir
+	return ..()
+
+/datum/status_effect/lwap_scope/tick()
+	locks = 0
+	var/turf/owner_turf = get_turf(owner)
+	var/scope_turf
+	for(var/turf/T in RANGE_EDGE_TURFS(7, owner_turf))
+		if(get_dir(owner, T) != owner_dir)
+			continue
+		if(T in range(owner, 6))
+			continue
+		scope_turf = T
+		break
+	if(scope_turf)
+		for(var/mob/living/L in range(10, scope_turf))
+			if(locks >= LWAP_LOCK_CAP)
+				return
+			if(L == owner)
+				continue
+			if(L.stat == DEAD)
+				continue
+			if(isslime(L) || ismonkeybasic(L)) //xenobio moment
+				continue
+			new /obj/effect/temp_visual/lwap_ping(owner.loc, owner, L)
+			locks++
+			continue
```

Comment:
```suggestion
```
dangling continue, loop will autocontinue once it reaches the end of the code block